games-action/cylindrix/cylindrix-1.0

```diff
@@ -1,7 +1,8 @@
 SUMMARY="A combat game"
 DESCRIPTION="Cylindrix is a 3 degrees of freedom combat game"
 HOMEPAGE="https://www.allegro.cc/depot/Cylindrix"
-COPYRIGHT="2001 John R. McCawley III, Anthony Thibault"
+COPYRIGHT="2001 John R. McCawley III
+	2001 Anthony Thibault"
```

Comment:
Should check on x86_64, but build is ok here on gcc2h secondary architecture without this change